Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -70,6 +70,8 @@ GameListItem::GameListItem(const std::string& _rFileName)
 	, m_BlobCompressed(false)
 	, m_ImageWidth(0)
 	, m_ImageHeight(0)
+	, m_disc_number(0)
+	, m_Country(DiscIO::IVolume::COUNTRY_UNKNOWN)
```

Comment:
m_Country needs to go above m_Valid, otherwise it'll cause warnings on Linux/OSX